core/infra/Dockerfile.tests

```diff
@@ -0,0 +1,8 @@
+FROM golang:alpine
+
+RUN apk add ffmpeg
```

Comment:
interesting, why do we need ffmpeg? does core require it?

Reply:
Forgot to remove it after copying the mediorum unittest dockerfile.